fix(tutorials): added tags corrected typos and fixed indentation MTA-1585 (#225)
scaleway/docs-content: tutorials/install-openvpn/index.mdx

  
  </Message>

> 6. Start the OpenVPN service. It is required to specify the configration file as an instance variable. The configuration file you created previously is called `/etc/openvpn/openvpn-server.conf`, this means you have to add **@openvpn-server** to the end of your unit when launching it:

Reviewer: ```suggestion
6. Start the OpenVPN service. It is necessary to specify the configuration file as an instance variable. The configuration file you created previously is called `/etc/openvpn/openvpn-server.conf`. Therefore, you need to add **@openvpn-server** to the end of your unit when launching it:
```

  Jan 21 14:43:35 openvpn ovpn-openvpn-server[30023]: Initialization Sequence Completed
  ```

> 8. Enable the service to start it automatically during the boot of your instance.

Reviewer: ```suggestion
8. Enable the service to start it automatically when your Instance boots.
```
